src/cli_shared/cli/mod.rs

```diff
@@ -231,6 +231,18 @@ impl CliOpts {
     }
 }
 
+/// CLI RPC options
+#[derive(Default, Debug, Parser)]
+pub struct CliRpcOpts {
+    /// Choose network chain to connect to
+    #[arg(long)]
+    pub chain: Option<NetworkChain>,
```

Comment:
This `chain` field is not needed. In the short-term, the wallet commands can query the network from the forest-node. In the long-term (when `forest-wallet` starts managing the keys separately from the node), we need to tag the keys with their testnet status.

Reply:
Ok, i was hesitant at removing it.